src/AlgebraicGeometry/Schemes/Divisors/WeilDivisor.jl

```diff
@@ -729,7 +734,7 @@ end
 
 Given an `AbsWeilDivisor` `D` on a scheme `X`, create a principal divisor 
 `div(f)` for some rational function, so that `D - div(f)` is supported 
-away from the original support of `D`.
+in (non-closed) scheme-theoretic points which are different from those of `D`.
```

Comment:
I still do not understand? The quantors are missing.  Are the supports disjoint? Or just not equal or ...?

Reply:
I hope that I have made it more clear in the latest version?